wildfly-modules/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM jboss/wildfly:latest
```

Comment:
Commenting now on the correct PR :package: 

You should never use `:latest` here. Please specify version here.